### ISSUE TITLE
Searches should be save-able by hitting return

### ISSUE THREAD

**Issue by waldoj** [flexion/hud-disaster-data#30]
It is necessary to select the floppy disk icon to save a search. For accessibility's sake, and consistency with the search field, the enter/return key should be bound to that form action.
